test/spec/app_spec.rb

```diff
@@ -433,25 +432,168 @@
+      element.click
+      input = @driver.find_element(:id, 'source__name-container')
+      input.send_keys(" - EDIT ACDC")
+      input = @driver.find_element(:id, 'source__bio-container')
+      input.send_keys(" - EDIT DESC")
+      @driver.find_element(:class, "source__edit-avatar-button").click
+      sleep 1
+      input = @driver.find_element(:css, 'input[type=file]')
+      input.send_keys(File.join(File.dirname(__FILE__), 'test.png'))
+      sleep 1
+      @driver.find_element(:class, 'source__edit-save-button').click
+      sleep 5 
+      displayed_name = get_element('h1.source__name').text
+      expect(displayed_name.include? "EDIT").to be(true)
+    end
 
-    # it "should add and remove accounts to sources" do
-    #   skip("Needs to be implemented")
-    # end
+    it "should add and remove accounts to sources", bin6: true  do
+      api_create_team_project_and_source_and_redirect_to_source('GOT', 'https://twitter.com/GameOfThrones')
+      sleep 5
+      element = @driver.find_element(:class, "source__edit-button")
+      element.click
+      sleep 3 
+      @driver.find_element(:class, "source__edit-addinfo-button").click
+      sleep 1
+      @driver.find_element(:class, "source__add-link").click
+      sleep 1
+      fill_field("source__link-input0", "www.acdc.com", :id)
+      sleep 2
+      @driver.find_element(:class, 'source__edit-save-button').click
+      sleep 5 
+      expect(@driver.page_source.include?('AC/DC Official Website')).to be(true)
+      #delete
+      element = @driver.find_element(:class, "source__edit-button")
+      element.click
+      sleep 3
+      list = @driver.find_elements(:css => "svg[class='create-task__remove-option-button create-task__md-icon']")
+      list[1].click
+      sleep 1
+      @driver.find_element(:class, 'source__edit-save-button').click
+      sleep 5 
+      expect(@driver.page_source.include?('AC/DC Official Website')).to be(false)
+    end
 
-    # it "should edit source metadata (contact, phone, location, organization, other)" do
-    #   skip("Needs to be implemented")
-    # end
+    it "should edit source metadata (contact, phone, location, organization, other)", bin6: true do    
+      api_create_team_project_and_source_and_redirect_to_source('GOT', 'https://twitter.com/GameOfThrones')
+      sleep 5
+      element = @driver.find_element(:class, "source__edit-button")
+      element.click
+      sleep 1 
+      @driver.find_element(:class, "source__edit-buttons-add-merge").click
+      sleep 1
+      @driver.find_element(:class, "source__add-phone").click
+      str= @driver.page_source
+      str = str[str.index('undefined-undefined-Phone-')..str.length]
+      str = str[0..(str.index('"')-1)] 
+      element = @driver.find_element(:id, str)
+      fill_field(str, "989898989", :id)
 
-    # it "should add and remove tags" do
-    #   skip("Needs to be implemented")
-    # end
+      sleep 1
+      @driver.find_element(:class, "source__edit-buttons-add-merge").click
```

Comment:
same here. this class should preferably be `source__edit-addinfo-button` instead of `source__edit-buttons-add-merge`